travis/travis_install_nightly

```diff
@@ -14,6 +14,14 @@ if [ "${LINT_CHECK}" != "0" ]; then
     exit 0
 fi
 
+if [ "${AUTODOC}" != "0" ]; then
+    pip install --upgrade sphinx sphinx-rtd-theme sphinx_bootstrap_theme odoo-sphinx-autodoc
+    pip install --upgrade docutils
+    pip install --upgrade pillow
+    pip install --upgrade --user travis-sphinx
+    export PATH=$HOME/.local/bin:$PATH
```

Comment:
I suggest placing the name of all packages above into a file `requirements-travis-sphinx.txt` and calling `pip install --upgrade -r requirements-travis-sphinx.txt`.

I don't think installing `travis-sphinx` with `--user` is necessary.